microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/advanced/HybridDocumentationTest.java

```diff
@@ -81,7 +81,6 @@
     @Deployment(name = PROVIDER_DEPLOYMENT_NAME, order = 1, testable = false)
     public static Archive<?> serviceProviderDeployment() {
         //  MP OpenAPI disabled on this Services Provider deployment for testing purposes, we don't want it here
```

Comment:
Since the property was moved to line 94, could this commentary also moved before line 94 in order to have a clear explanation why we disabling this property?

Reply:
Yes, definitely, thanks.